src/main/java/ai/mindgard/MindgardSettingsManager.java

```diff
@@ -1,18 +0,0 @@
-package ai.mindgard;
-/**
- * Mindgard Settings Manager
- * Used to synchronise both the MindgardSettingsUI and MindgardSettings views of the state of the settings.
- */
-public class MindgardSettingsManager {
```

Comment:
`currentSettings` private internal state; change setSettings/getSettings to set/get from this. Unsure whether this can be static or whether we intantiate the manager and pass that around.